source/backend/api/Pims.Api.csproj

```diff
@@ -7,6 +7,8 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <ProjectGuid>{16BC0468-78F6-4C91-87DA-7403C919E646}</ProjectGuid>
     <TargetFramework>net8.0</TargetFramework>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
```

Comment:
The default GC setting is actually "workstation" which doesn't seem appropriate for our case. See here: 
https://learn.microsoft.com/en-us/dotnet/standard/garbage-collection/workstation-server-gc